internal/core/src/segcore/segcore_init_c.cpp

```diff
@@ -9,15 +9,17 @@
 // is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 // or implied. See the License for the specific language governing permissions and limitations under the License
 
-#include "index/thirdparty/faiss/FaissHook.h"
-#include "segcore/segcore_init_c.h"
-#include "knowhere/archive/KnowhereConfig.h"
 #include <iostream>
+
+#include "exceptions/EasyAssert.h"
+#include "knowhere/archive/KnowhereConfig.h"
+#include "segcore/segcore_init_c.h"
+#include "segcore/SegcoreConfig.h"
 #include "utils/Log.h"
 
 namespace milvus::segcore {
 static void
-SegcoreInitImpl() {
+SegcoreInitImpl(const char* config_dir) {
```

Comment:
Split the configuration part into different pr

Reply:
done